### PR TITLE
Html report image embedding and jasper report update

### PR DESCRIPTION
**Pull request by Gothami**

### CODE REVIEW

**Comment by Nipuni** [comment]
This need to be reviewed by the team lead

**Comment by cnapagoda** [comment]
Reviewed, please merge

**Comment by Gothami** [comment]
This PR is for [1]
[1]https://wso2.org/jira/browse/CCOMMONS-3

**Comment by Gothami** [comment]
The integration test can be found in [1].

[1]https://github.com/Gothami/product-greg/blob/feature/gothami-test/modules/integration/tests-integration/tests-extensibility/tests-report/src/test/java/org/wso2/carbon/registry/reporting/test/ReportGenerationImageConversionTest.java

File: components/reporting/org.wso2.carbon.reporting.util/src/main/java/org/wso2/carbon/reporting/util/types/HtmlReport.java [hunk 1]
```diff
@@ -37,40 +41,38 @@
      * @return reporting ByteArrayOutputStream
      * @throws ReportingException when JasperPrint null
      * @throws IOException
-     * @throws JRException
      */
     public ByteArrayOutputStream generateHtmlReport(JasperPrint jasperPrint)
-            throws ReportingException, JRException {
+            throws ReportingException{
```
Comment (Nipuni) [inline components/reporting/org.wso2.carbon.reporting.util/src/main/java/org/wso2/carbon/reporting/util/types/HtmlReport.java]: formatting issue

**Pull request closed without merging**